gvr-javascript/assets/script.js

```diff
@@ -1,30 +1,43 @@
+importPackage(org.gearvrf)
+importPackage(org.gearvrf.scene_objects)
+
 function onInit(gvrf) {
   var mainScene = gvrf.getNextMainScene();
 
-  // 3D model
-  var model = gvrf.loadModel("astro_boy.dae");
-  model.getTransform().setRotationByAxis(45.0, 0.0, 1.0, 0.0);
-  model.getTransform().setScale(3, 3, 3);
-  model.getTransform().setPosition(0.0, -0.4, -0.5);
-
-  mainScene.addSceneObject(model);
+  // 3D Boat Scene
+  var boat = gvrf.loadModel("RowBoatFBX/RowBoatAnimated.fbx");
+  boat.setName("boat");
+  boat.getTransform().setScale(.5, .5, .5);
+  boat.getTransform().setRotationByAxis(20, 0, 1, 0);
+  boat.getTransform().setPosition(20, -20, -40);
+  mainScene.addSceneObject(boat);
 
   // Text
-  var textView = utils.newTextViewSceneObject(gvrf, "GVRf scripting in Javascript");
-  var textSize = textView.getTextSize();
-  textView.setTextSize(textSize);
-  textView.getTransform().setPosition(0, 0, -2);
+  var textView = new GVRTextViewSceneObject(gvrf);
+  textView.setText("GVRf scripting in Javascript");
+  textView.setRefreshFrequency(GVRTextViewSceneObject.IntervalFrequency.HIGH);
+  textView.setName("text");
+  textView.setTextSize(50);
+  textView.getTransform().setPosition(0, 0, -100);
+  textView.getTransform().setScale(50, 50, 50);
+
+  var textSensor = new GVRBaseSensor(gvrf);
+  textView.setSensor(textSensor);
 
   mainScene.addSceneObject(textView);
 
   // Animation
   var animationEngine = gvrf.getAnimationEngine();
-  var animations = utils.getAnimations(model);
-  if (animations.length != 0) {
-      animations[0].setRepeatMode(1).setRepeatCount(-1);
-      animations[0].start(animationEngine);
+  var animations = boat.getAnimations();
+  if (animations.size() != 0) {
+      animations.get(0).setRepeatMode(1).setRepeatCount(-1);
+      animations.get(0).start(animationEngine);
   }
+
+  mainScene.setFrustumCulling(false);
```

Comment:
Thanks for asking. Animation will change the bounding box and culling currently doesn't work with that.